.circleci/config.yml

```diff
@@ -320,6 +320,12 @@ workflows:
           test_tags: "recover"
           requires:
             - packet-deploy-k8s-2
+      - packet-integration-tests:
+          name: "test benchmark"
+          cluster_id: "1"
+          test_tags: "bench"
+          requires:
+            - packet-deploy-k8s-1
```

Comment:
Like this the CI will try to run in parrallel "test basic usecase" and "test basic usecase".
Better to put here 
```yaml
      - packet-integration-tests:
	          name: "test benchmark"
	          cluster_id: "2"
	          test_tags: "bench"
	          requires:
	            - packet-deploy-k8s-1
	            - "test recover"
```

Ther recover tests are shorter, so queuing the benchmark on the same cluster just after them makes sense.

Reply:
thanks, fixed.

Reply:
Sorry, I should have been more accurate, you actually need:
```yaml
      - packet-integration-tests:
	          name: "test benchmark"
	          cluster_id: "2"
	          test_tags: "bench"
	          requires:
	            - "test recover"
```

Reply:
thanks, fixed